code/modules/mob/living/basic/basic_defense.dm

```diff
@@ -22,29 +22,30 @@
 		return TRUE
 
 	if(!user.combat_mode)
-		if (stat == DEAD)
-			return
-		visible_message(span_notice("[user] [response_help_continuous] [src]."), \
-						span_notice("[user] [response_help_continuous] you."), null, null, user)
-		to_chat(user, span_notice("You [response_help_simple] [src]."))
-		playsound(loc, 'sound/weapons/thudswoosh.ogg', 50, TRUE, -1)
-	else
-		if(HAS_TRAIT(user, TRAIT_PACIFISM))
-			to_chat(user, span_warning("You don't want to hurt [src]!"))
-			return
-		user.do_attack_animation(src, ATTACK_EFFECT_PUNCH)
-		visible_message(span_danger("[user] [response_harm_continuous] [src]!"),\
-						span_userdanger("[user] [response_harm_continuous] you!"), null, COMBAT_MESSAGE_RANGE, user)
-		to_chat(user, span_danger("You [response_harm_simple] [src]!"))
-		playsound(loc, attacked_sound, 25, TRUE, -1)
-		var/obj/item/bodypart/arm/active_arm = user.get_active_hand()
-		var/damage = (basic_mob_flags & IMMUNE_TO_FISTS) ? 0 : rand(active_arm.unarmed_damage_low, active_arm.unarmed_damage_high)
-
-		attack_threshold_check(damage)
-		log_combat(user, src, "attacked")
-		updatehealth()
+		if (stat != DEAD)
+			visible_message(span_notice("[user] [response_help_continuous] [src]."), \
+							span_notice("[user] [response_help_continuous] you."), null, null, user)
```

Comment:
```suggestion
			visible_message(
				message = span_notice("[user] [response_help_continuous] [src]."), \
				self_message = span_notice("[user] [response_help_continuous] you."),
				ignored_mobs = list(user),
			)
```

A) no need to forcefeed two nulls in just so we can use positional argument (also named is nice :3)
B) indentation is better
C) `ignored_mobs` is a list argument so let's fix that while we are here (yes i know `visible_message()` will fix it for you but let's be clean dammit)
D) you already were multilining visible message calls so let's do this one too.